day_01/day_01.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+import argparse
+import pathlib
+import re
+
+import day_01
+
+DEFAULT_DATA = pathlib.Path(day_01.__file__).parent.parent / "data"/ "day1.txt"
+
+
+def load(filename: str | pathlib.Path) -> list[str]:
```

Comment:
Moved actions into functions to make it reusable in unit tests or other parts - specify the filename.
In future days it may be interesting to load the data in a more complex manner to store it directly in the appropriate structure.
For this one a list is good.